app/templates/_webpack.config.js

```diff
@@ -3,31 +3,38 @@ import webpack from 'webpack';
 import UglifyJsPlugin from 'uglifyjs-webpack-plugin';
 import config from './<%= answers.projectName %>.Gulp/config';
 
+const production = process.env.NODE_ENV === 'production';
 const bundleFilename = '[name].bundle.js';
-const chunkFilename = '[name].[chunkhash].chunk.js';
-const vendorRegex = /node_modules/;
 
 export default {
-    mode: global.production ? 'production' : 'development',
+    mode: production ? 'production' : 'development',
     entry: {
         '<%= answers.projectName %>': `${config.js.src}/<%= answers.projectName %>/index.js`
     },
     output: {
         path: path.resolve(__dirname, config.js.dest),
         publicPath: `${config.js.dest.replace(config.baseDir, '')}/`,
         filename: bundleFilename,
-        chunkFilename: chunkFilename
+        chunkFilename: '[name].[chunkhash].chunk.js'
+    },
+    devtool: production ? 'source-map' : 'cheap-module-eval-source-map',
+    resolve: {
+        alias: {
+            '@': path.resolve(__dirname, config.js.src)
```

Comment:
While it seems like a nice idea to use `@` instead of `../../../`, it breaks autocomplete and other useful IDE functions. How do you deal with this stuff?

Reply:
Well, I use VS code and usually add [jsconfig.json](https://code.visualstudio.com/docs/languages/jsconfig) to support path resolving along with  other things such as excluding `node_modules` folder. Now we could add it to the generator but wasn't sure if it belonged there. 🎲

Reply:
Clever! 👍 Thanks, didn't know that 😄